mobile/test/swift/EngineBuilderTests.swift

```diff
@@ -425,18 +425,31 @@ final class EngineBuilderTests: XCTestCase {
     XCTAssertTrue(bootstrapDebugDescription.contains("FAKE_SWIFT_ADDRESS"))
   }
 
+  func testAddingCdsAndAdsConfigurationWhenRunningEnvoy() {
+    let bootstrapDebugDescription = EngineBuilder()
+      .addEngineType(MockEnvoyEngine.self)
+      .addCDSLayer(cdsResourcesLocator: "FAKE_CDS_LOCATOR", timeoutSeconds: 2543)
+      .setAggregatedDiscoveryService(address: "FAKE_SWIFT_ADDRESS", port: 0)
+      .bootstrapDebugDescription()
+    XCTAssertTrue(bootstrapDebugDescription.contains("FAKE_CDS_LOCATOR"))
+    XCTAssertTrue(bootstrapDebugDescription.contains("initial_fetch_timeout { seconds: 2543 }"))
+    XCTAssertTrue(bootstrapDebugDescription.contains("FAKE_SWIFT_ADDRESS"))
```

Comment:
perhaps another test for default values for cdsResourcesLocator and timeout, similar to the suggestion for Kotlin